### PR TITLE
Fix All-Namespaces Flag

### PR DESCRIPTION
**Pull request by alecmerdler**
### Description

Running with `--namespaces=""` should run in against all namespaces in the cluster. This is what OLM will pass to the Operator when deployed in a "global" `OperatorGroup` (recommended). Changes the usage of this flag from 
```
--namespace=default --namespace=kube-system
``` 
to 
```
--namespaces=default,kube-system
```

### CODE REVIEW

File: cmd/security-labeller/main.go [hunk 1]
```diff
@@ -36,7 +37,7 @@ func main() {
 	flag.CommandLine = flag.NewFlagSet(os.Args[0], flag.ExitOnError)
 
 	var namespaces arrayFlags
-	flag.Var(&namespaces, "namespace", "Namespace to scan.")
+	flag.Var(&namespaces, "namespaces", "Namespaces to scan, separated by commas. Leave empty to scan all namespaces.")
```
Comment (kleesc) [inline cmd/security-labeller/main.go]: Actually, can we specify namespaces separated by commas? Won't that set the element in the array to `"namespace1,namespace2"`.
My understanding was that the user would specify multiple namespace: `--namespace namespace1 --namespace namespace2` and that not specifying `--namespace` at all would give the `NamespaceAll`

File: cmd/security-labeller/main.go [hunk 2]
```diff
@@ -62,6 +63,9 @@ func main() {
 			panic(err)
 		}
 	} else {
+		if len(namespaces) == 1 && namespaces[0] == corev1.NamespaceAll {
+			namespaces = []string{}
+		}
```
Comment (kleesc) [inline cmd/security-labeller/main.go]: Won't `namespaces` be empty if no `--namespace` is specified?

**Comment by alecmerdler** [comment]
We can push the work to OLM packaging and just use Bash to pass the needed `--namespace` flags using the `WATCH_NAMESPACE` environment variable for the pod.

File: cmd/security-labeller/main.go (hunk 1 above)
Comment (kleesc) [reply cmd/security-labeller/main.go]: This should be a `flag.String` then.

Comment (alecmerdler) [reply cmd/security-labeller/main.go]: We can still use `flag.Var`, just needed to change the `Set()` method.

**Pull request merged**